build/updater.sh

```diff
@@ -2,6 +2,7 @@
 cd /data/project/query-builder-test/query-builder
 git reset --hard origin/master
 git pull
+export NODE_ENV=production
 npm install npm@6.14.8
```

Comment:
I assume we need to inject values of .env.production here too. Right? or do I need more coffee?

Reply:
lol no :)
once the NODE_ENV is set to production, Vue will automatically pick up the values from .env.production file